draft-thomson-httpbis-http2bis.xml

```diff
@@ -1167,231 +1162,77 @@
-            assigned a <xref target="pri-default">default priority</xref>.  This potentially creates
-            suboptimal prioritization, since the stream could be given a priority that is different
-            from what is intended.
-          </t>
-          <t>
-            To avoid these problems, an endpoint SHOULD retain stream prioritization state for a
-            period after streams become closed.  The longer state is retained, the lower the chance
-            that streams are assigned incorrect or default priority values.
-          </t>
+        <section anchor="PriorityHere">
+          <name>Priority Signaling in this Document</name>
           <t>
-            Similarly, streams that are in the "idle" state can be assigned priority or become a
-            parent of other streams.  This allows for the creation of a grouping node in the
-            dependency tree, which enables more flexible expressions of priority.  Idle streams
-            begin with a <xref target="pri-default">default priority</xref>.
+            This update to HTTP/2 deprecates the priority signaling defined in <xref
+            target="RFC7540">RFC 7540</xref>.  The bulk of the text related to priority signals is
+            not included in this document.  The description of frame fields and some of the
+            mandatory handling is retained to ensure that implementations of this document remain
+            interoperable with implementations that use the priority signaling described in RFC
+            7540.
           </t>
           <t>
-            The retention of priority information for streams that are not counted toward the limit
-            set by <xref target="SETTINGS_MAX_CONCURRENT_STREAMS" format="none">SETTINGS_MAX_CONCURRENT_STREAMS</xref> could create a large state burden
-            for an endpoint.  Therefore, the amount of prioritization state that is retained MAY be
-            limited.
+            A thorough description of the RFC 7540 priority scheme remains in <xref target="RFC7540"
+            section="5.3"/>.
           </t>
           <t>
-            The amount of additional state an endpoint maintains for prioritization could be
-            dependent on load; under high load, prioritization state can be discarded to limit
-            resource commitments.  In extreme cases, an endpoint could even discard prioritization
-            state for active or reserved streams. If a limit is applied, endpoints SHOULD maintain
-            state for at least as many streams as allowed by their setting for
-            <xref target="SETTINGS_MAX_CONCURRENT_STREAMS" format="none">SETTINGS_MAX_CONCURRENT_STREAMS</xref>.  Implementations SHOULD also attempt to
-            retain state for streams that are in active use in the priority tree.
+            Signaling priority information could still be necessary to attain good performance.
```

Comment:
Above, it is said that good prioritization can be critical and poor prioritization can lead to performance degradation, which I agree with. As such "could still be necessary" is too weak of a statement. "is still an important aspect to attaining good HTTP/2 performance" would be better.

Reply:
Done.  Thanks.